src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,8 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService user = new UserService();
```

Comment:
1. bad var name for user service
2. declare the service as class level field
```suggestion
        UserService user = new UserService();
```

Reply:
@sokimaaa updated